client-cli/src/command.rs

```diff
@@ -219,10 +220,30 @@ pub enum Command {
             name = "light client peer",
             short = "l",
             long = "light-client-peers",
-            default_value = "",
             help = "Light client peers"
         )]
-        light_client_peers: String,
+        light_client_peers: Option<String>,
+
+        #[structopt(
+            name = "light client trusting period in seconds",
+            long = "light-client-trusting-period",
+            help = "light client trusting period in seconds"
+        )]
+        light_client_trusting_period_seconds: Option<u64>,
```

Comment:
not sure if this should be settable or should always equal to maximum evidence age?

Reply:
@yihuang

Reply:
I think no need to set this by user?

Reply:
in testing environment, i setup that period very small, (10 seconds)
currently, default value is evidence age

Reply:
User might want to choose a trusting period smaller than unbonding period(max evidence age) for better safety.

Reply:
ya, agree, 
i'll add display max evidence age